cmd/protoc-gen-connect-go/main.go

```diff
@@ -19,23 +19,23 @@
 // The 'connect-go' suffix becomes part of the arguments for the Protobuf
 // compiler. To generate the base Go types and Connect code using protoc:
 //
-//	 protoc --go_out=gen --connect-go_out=gen path/to/file.proto
+//	protoc --go_out=gen --connect-go_out=gen path/to/file.proto
 //
 // With buf, your buf.gen.yaml will look like this:
 //
-//   version: v1
-//   plugins:
-//     name: go
-//     out: gen
-//     name: connect-go
-//     out: gen
+//	version: v1
+//	plugins:
+//	  name: go
```

Comment:
Shouldn't there be a '-' before each name to separate plugins?

Reply:
🤦🏽‍♂️ Yep!